### PR TITLE
Make $emailAddress on TypeException to be evaluated

### PR DESCRIPTION
**Pull request by yehudah**
- [X ] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [X ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the development branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

Change the exception message from single quotes to double quotes so the user will know which email address failed.

### CODE REVIEW

**Comment by yehudah** [comment]
There is another issue on the code flow, that I didn't have the time to check.

I have this code:

```
$bccEmails = array();
foreach ( ( array ) $message->getBccRecipients() as $recipient ) {
                $recipient->log($this->logger, 'Bcc');
                $bccEmails[] = new \SendGrid\Mail\Bcc( $recipient->getEmail(), $recipient->getName() );
}
$email->addBccs($bccEmails);
```

By mistake instead, to call The Bcc class I called the Cc class, which caused the $emailAddress to be SendGridMailCc Object and not a string like the documentation says.

**Comment by thinkingserious** [comment]
Hello @yehudah,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @yehudah,

Thank you for the PR!

With regards to your second issue with respect to code flow, I'm not quite sure I follow. What is the expected result?

With Best Regards,

Elmer

File: lib/mail/EmailAddress.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function setEmailAddress($emailAddress)
             filter_var($emailAddress, FILTER_VALIDATE_EMAIL))
         ) {
             throw new TypeException(
-                '$emailAddress must be valid and of type string.'
+                "{$emailAddress} must be valid and of type string."
```
Comment (rvitaliy) [inline lib/mail/EmailAddress.php]: keep attention if a `$emailAddress` is not a string and it's can't be converto to string this generate a php error!
if you need to cast this value to string, probably the best choise is do something like: https://github.com/webmozart/assert/blob/master/src/Assert.php#L1145

**Pull request merged**